test/Tool/known6.opt

```diff
@@ -0,0 +1,11 @@
+; REQUIRES: solver
+
+; RUN: %souper-check %solver -print-counterexample=false  %s > %t 2>&1
+; RUN: FileCheck %s < %t
+
+; CHECK: LGTM
+%0:i8 = var (0000000x)
```

Comment:
This test also does not depend on the known bits.

Please go through all of these test cases. For each one, make sure that the test case actually tests what you are intending to test. Unit tests should not be random but should have a point.